session/scripts/script.js

```diff
@@ -0,0 +1,5 @@
+function myFunction() {
+    alert("Wrong Username and/or password");
+    window.location.href = "/";
+}
+myFunction();
```

Comment:
This red symbol that you can see at the bottom, that is coming because of absence of newline at the end of file.
So please add an empty line at the end of file.
```
<?php
  -------------
  ---------------
  $abc = '1';

```